integration/binding_with_optional_field/strings/strings.to_upper.go

```diff
@@ -0,0 +1,22 @@
+// Code generated by protoc-gen-goclay, but your can (must) modify it.
+// source: pb/strings.proto
+
+package strings
+
+import (
+	"context"
+	desc "github.com/utrack/clay/integration/binding_with_optional_field/pb"
+	"strings"
+)
+
+func (i *StringsImplementation) ToUpper(ctx context.Context, req *desc.String) (rsp *desc.String, err error) {
+	rsp = &desc.String{}
+	if nil == req.Str {
```

Comment:
It looks a bit more go way to me
```suggestion
	if req.Str == nil {
```

Reply:
Thank you